src/modules/dashboard/components/Dialogs/AddExistingSafeDialog/ConnectSafe.tsx

```diff
@@ -62,25 +91,54 @@ const ConnectSafe = ({
   values: { chainId },
   setStepIndex,
   colonyAddress,
+  safeAddress,
 }: ConnectSafeProps & FormikProps<FormValues>) => {
+  const selectedChain = SUPPORTED_SAFE_NETWORKS.find(
+    (network) => network.chainId === Number(chainId),
+  );
+  const { formatMessage } = useIntl();
+  const moduleHref = `https://app.safe.global/${selectedChain?.shortName.toLowerCase()}:${safeAddress}/apps?appUrl=https%3A%2F%2Fzodiac.gnosisguild.org%2F`;
```

Comment:
Perhaps move this to a helper function like `getModuleLink()` that returns this href? I think that would be nicer to read, especially if you're just skimming through. 

And perhaps assign the constants that prepend and append the selectedChain/safeAddress to variables in `externalUrls` also?

Reply:
I don't think putting this inside a function will make it more readable, however, I do have some alternatives in mind now that you have mentioned `externalUrls`